<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="3,2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,2">
       <item>
        <widget class="QLabel" name="cameraView">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>选择相机：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="camraList">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>新宋体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>选择声卡：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="audioList">
             <property name="font">
              <font>
               <family>新宋体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>摄像头选项：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cameraInfo">
           <property name="font">
            <font>
             <family>新宋体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="start_record">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <family>新宋体</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>开始</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stop_record">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <family>新宋体</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>停止</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QtCharts::QChartView" name="audioView">
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
